apps/storefront-e2e/src/integration/checkout.cy.ts

```diff
@@ -66,6 +71,30 @@ describe('Checkout suite', () => {
           .and('contain', 'Billing address')
           .and('contain', 'Delivery address');
 
+        thankYouPage.getOrderEntriesWrapper().should('be.visible');
+        thankYouPage.getOrderEntries().should('have.length.at.least', 1);
+
+        thankYouPage.orderTotalsFragment.getWrapper().should('be.visible');
```

Comment:
Can we place the `checkTotals` function in `TotalFragment` or in separate `totals.actions.ts` because it isn't only for the cart process but also for the checkout?

Reply:
Lets put it in `TotalFragment`, should be fine